test/e2e/network/netpol/model.go

```diff
@@ -21,11 +21,20 @@ import (
 	"strings"
 
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/kubernetes/test/e2e/framework"
 	imageutils "k8s.io/kubernetes/test/utils/image"
 )
 
+// getResourceRequirements returns a ResourceRequirements object
```

Comment:
... add note about how this was borrowed from another place in the code base, or else, see if we can reuse the original code that i copied this from :)

Reply:
i have pushed a commit with a note about where this was borrowed from, I don't think we can import it from the code base. A quick search https://cs.k8s.io/?q=getResourceRequirements&i=nope&files=&excludeFiles=&repos= shows that this function is defined in other parts of codebase the way we have done it

lmk if you think otherwise